locales/ar-arb/messages.po

```diff
@@ -779,15 +783,11 @@ msgid "Lower than or equal"
 msgstr "أقل من أو يساوي"
 
 msgid "Malformed XML[%s]:\n"
```

Comment:
I think we have missed here a line. Please, look at line 782 on the source `"%s"`. It has been removed in the translation, right?

Reply:
Reverted :)